index.html

```diff
@@ -105,6 +105,31 @@
 </p>
 
 <!--
+  ACCESSIBILITY
+
+  Modify the block below to indicate what kinds of accessibility are available
+  for you workshop. If you cannot provide a particular kind of accessibility,
+  just clearly state in the paragraph below that it is not available.
+
+  If you're not sure what to include here see:
+  http://software-carpentry.org/workshops/checklists/accessibility.html Your
+  university's disability resource center should be able to help you figure out
+  what kinds of accessibility are available through the university.
+-->
+
+<p>
+  <strong>Accessibility:</strong>The building for this workshop meets all
+  Americans with Disabilities Act (ADA) physical facilities requirements. There
```

Comment:
Given how many workshops we run outside the US, is there wording that will internationalize?